plugins/completion/cmp/default.nix

```
@@ -16,6 +16,43 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   maintainers = [ maintainers.GaetanLepage ];
 
+  description = ''
+    ### Completion Source Installation
+
+    If `plugins.cmp.autoEnableSources` Nixivm will automatically enable the corresponding source
```

I think we should be more explicit than "if \<option\>", "if \<option\> is \<value\>" feels better.

I also think we should somehow indicate that this is the default state, however my wording below is kinda clunky, and I'm not happy with it :sweat: 
```suggestion
    If `plugins.cmp.autoEnableSources` is true, which it is by default, Nixivm will automatically enable the corresponding source
```